guides/09_Amazon_Voice_Focus.md

```diff
@@ -283,9 +283,27 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 ## Enabling Voice Focus with Echo Reduction
 
-Enabling Voice Focus with Echo Reduction is a two step process: 
+Enabling Voice Focus with Echo Reduction is a three step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Changing the namespace to Amazon Chime Meetings SDK
```

Comment:
This step is to create meeting so I think the original title is fine. How about `Create a meeting with support for Echo Reduction using Amazon Chime Meeting SDK namespace` ?

Reply:
The original portion is still a part of the documentation. It has just become step 2 instead of being step 1

Reply:
They are related so shouldn't be the same step?

Reply:
They can be. Let me change it